### PR TITLE
Add height/width kwarg support to bokeh.io.export

### LINKED ISSUE #6369: Add height/width kwargs to bokeh.io.export to support resizing the exported PNG
Add height/width kwargs to bokeh.io.export to support resizing the exported PNG

### PR DESCRIPTION
**Pull request by canavandl**
- [x] issues: fixes #6369 
- [x] tests added / passed
- [ ] release document entry (if new feature or API change)

### CODE REVIEW

File: bokeh/io.py [hunk 1]
```diff
@@ -652,6 +667,9 @@ def export(obj, filename=None):
     '''
     image = _get_screenshot_as_png(obj)
 
+    if height is not None or width is not None:
+        image = _resize_image(height=height, width=width)
```
Comment (mattpap) [inline bokeh/io.py]: `_resize_image()` has different signature (`image`).

Comment (canavandl) [inline bokeh/io.py]: fixed

**Comment by bryevdv** [comment]
@canavandl I think this may need some consideration. I think we need to do more than call an image resize after the fact. I think we should try to generate images of the right size from the start. e.g. we could tinker with plot sizes before running through the export machinery.

**Comment by bryevdv** [comment]
Will revisit in another PR

**Pull request closed without merging**